sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
```

Comment:
Did you run the script to update these? The dependency version looks like 1.1.0. @JimSuplizio 
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L9

Reply:
Yeah, it looks like you'd updated the dependency version of azure-core-http-netty without checking in the version_client.txt.

Reply:
> HOWEVER, I don't think that this is the best long term solution. We need to come up with a more nuanced `update_versions.py` script which will only append the dev package suffix if the version that is already in the dependencies list matches what is defined in `versions_client.txt` (for example).

@mitchdenny by "update_versions.py" you mean "set_versions.py" don't you?

I believe that the problem problem is this: You're calling set_versions.py with "--update-type all" and you should really be calling it with "--update-type library". "-all" means update both the dependency and the current. You're creating versions to upload into the dev feed so only the current version needs to be changed and changing the dependency version isn't necessary.

Reply:
No I mean update versions. The problem occurs when one of the downstream dependents is relying on the version of core in the repository. When we change that version via update versions (with just - -ut library) it can no longer resolve that version. We either need to flick that down stream dependent to use a dev package or we have to update the whole graph.

Reply:
Remember that this update is scoped to just the build, it does t get checked in.

Reply:
For now I've just run ```update_versions.py --ut all``` across the repo to snap all the versions to where they need to be - this fixes dev package production. So once I put the condition back in I can merge this PR. I think that there might end up being some nuanced scenarios we need to deal with around versioning when a downstream dependency of core is no longer being worked on and we just leave its dependency version alone.

Thoughts?